docs/source/torch.compiler_custom_backends.rst

```diff
@@ -84,7 +84,7 @@ Registration serves two purposes:
 
 * You can pass a string containing your backend function's name to ``torch.compile`` instead of the function itself,
   for example, ``torch.compile(model, backend="my_compiler")``.
-* It is required for use with the `minifier <https://pytorch.org/docs/main/compile/troubleshooting.html>`__. Any generated
+* It is required for use with the `minifier <https://pytorch.org/docs/main/torch.compiler_troubleshooting.html>`__. Any generated
```

Comment:
These links inside `rst` files, should they not be using [cross-references](https://www.sphinx-doc.org/en/master/usage/referencing.html)? I presume that would avoid dead links in future.

Reply:
Agreed.  @svekars could you or anyone from Meta fix this one?